srcpkgs/libphonenumber/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'libphonenumber'
+pkgname=libphonenumber
+version=8.12.23
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config protobuf ninja"
+makedepends="boost-devel gtest-devel icu-devel protobuf-devel"
+short_desc="Google's common library for handling international phone numbers"
+maintainer="Louis Dupré Bertoni <contact@louisdb.xyz>"
+license="Apache-2.0"
+homepage="https://github.com/googlei18n/libphonenumber"
+distfiles="https://github.com/google/libphonenumber/archive/refs/tags/v${version}.tar.gz"
+checksum=49aed4e11eecff703207ad46f40cdb5995eece24d0f7e9aa25537aef5f011e51
+
+do_configure() {
+	cmake -S cpp -B build \
```

Comment:
Do not override do_configure/do_build/do_install/do_check, you can add what is needed via configure_args ( but you probably only need to set `build_wrksrc=cpp` )

Reply:
I originally tried it like that but it fails:

```
CMake Error:
  Running

   '/usr/bin/ninja' '-C' '/builddir/libphonenumber-8.12.23/cpp/build' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:1433: multiple rules generate ../src/phonenumbers/metadata.h [-w dupbuild=err]

CMake Generate step failed.  Build files cannot be regenerated correctly.
```

Reply:
set `make_cmd=make` and `export CMAKE_GENERATOR="Unix Makefiles"`

Reply:
thanks for the help :)